src/main/java/API_hw2/core/TrelloServiceObj.java

```diff
@@ -0,0 +1,183 @@
+package API_hw2.core;
+
+import API_hw2.beans.TrelloBoard;
+import API_hw2.data.BoardConstant;
+import API_hw2.data.ResponseStatus;
+import API_hw2.data.URLConstant;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import static io.restassured.http.ContentType.*;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.apache.http.HttpStatus;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.Properties;
+
+import static org.hamcrest.Matchers.*;
+
+public class TrelloServiceObj {
+
+    public static String URL;
+    private final Method requestMethod;
+
+    private final Map<String, String> parameters;
+
+    private final String pathToProperties = "src/main/resources/API_hw2/credentials.properties";
```

Comment:
A constant should be private static final + UPPER case

Reply:
fixed